app/services/IdentityService.scala

```diff
@@ -22,6 +24,51 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.language.{higherKinds, implicitConversions}
 import scalaz.{Monad, NonEmptyList, \/}
 
+// identity-play-auth (which previously provided CookieBuilder) has been deprecated in favour of identity-auth-play:
+// - old library: https://github.com/guardian/identity-play-auth
+// - new library: https://github.com/guardian/identity/pull/1571
+//
+// As part of this migration, only the key features of identity-play-auth were kept
+// i.e. authenticating a user from a Play RequestHeader
+//
+// Therefore, to preserve the CookieBuilder functionality,
+// the code has been copy and pasted from the old identity-play-auth.
+object CookieBuilder {
+
+  def cookiesFromDescription(
+    cookieDescriptionList: CookieDescriptionList,
+    domain: Option[String] = None
+  )(implicit clock: Clock = Clock.systemUTC()): Seq[Cookie] = {
+
+    val maxAge = Duration.between(clock.instant(), cookieDescriptionList.expiresAt).getSeconds.toInt
+
+    for (cookieDescription <- cookieDescriptionList.values) yield {
```

Comment:
what does `description` mean? could this be just `cookie`?  as it seems toinlcude the key as well

Reply:
or `identityCookie` if it's the one for the ideneity side rather than the play side

Reply:
Code was (mainly) copy and pasted from identity-play-auth - presumably the names were influenced by the types of the values they refer to 🤷‍♂️ 

I think given the names of the types, the variable names are reasonable, but they could be another change to make in (c.f.) identity 4.x

I'll put a comment though to make the code clearer.